code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -690,29 +690,27 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 	density = TRUE
 
 /obj/structure/table_frame/abductor/attackby(obj/item/I, mob/user, params)
-	if(istype(I, /obj/item/stack/sheet/mineral/abductor))
-		var/obj/item/stack/sheet/P = I
-		if(P.get_amount() < 1)
-			to_chat(user, "<span class='warning'>You need one alien alloy sheet to do this!</span>")
-			return
-		to_chat(user, "<span class='notice'>You start adding [P] to [src]...</span>")
-		if(do_after(user, 50, target = src))
-			P.use(1)
-			new /obj/structure/table/abductor(loc)
-			qdel(src)
+	if(!(istype(I, /obj/item/stack/sheet/mineral/abductor) || istype(I, /obj/item/stack/sheet/mineral/silver)))
+		return ..()
+
+	var/obj/item/stack/stack = I
+
+	if(stack.get_amount() < 1)
```

Comment:
what happens if you hit the table_frame with an item that's not a stack

Reply:
Line 693, it early returns and hits the table